cli/commands.go

```diff
@@ -284,25 +279,40 @@ func CreateCluster(c *cli.Context) error {
 
 	/* (2)
 	 * Server
-	 * Create the server node container
+	 * Create the server node containers
 	 */
-	serverContainerID, err := createServer(clusterSpec)
-	if err != nil {
-		deleteCluster()
-		return err
+	serverContainersIDs := []string{}
+	for i := 0; i < c.Int("servers"); i++ {
+		if i == 0 {
+			clusterSpec.ServerArgs = append(clusterSpec.ServerArgs, "--cluster-init")
+		} else {
+			masterServerIP, err := getContainerIP(serverContainersIDs[0])
+			if err != nil {
+				return err
+			}
+			serverAddressArg := fmt.Sprintf("--server=https://%s:%s", masterServerIP, apiPort.Port)
+			clusterSpec.ServerArgs = append(clusterSpec.ServerArgs, serverAddressArg)
+		}
+		serverContainerID, err := createServer(clusterSpec, i)
+		if err != nil {
+			deleteCluster()
+			return err
+		}
+		serverContainersIDs = append(serverContainersIDs, serverContainerID)
+		if err := waitForContainerLogMessage(serverContainerID, "Wrote kubeconfig", 120); err != nil {
```

Comment:
I figured out, that just 1 second delay between starting the server nodes can be enough to avoid bootstrapping issues (which occur, when two servers try to register with the init server at the same time)